src/social-network/defaults/injectPostInspector.tsx

```diff
@@ -24,7 +24,8 @@ export function injectPostInspectorDefault(config: InjectPostInspectorDefaultCon
         )
     }
 }
-interface InjectPostInspectorDefaultConfig {
+
+export interface InjectPostInspectorDefaultConfig {
```

Comment:
Why exporting this interface?

Reply:
fixed f1e9b52133bbd4989213a0d6c104084ea4b6c0ef